src/procedures/procedures.11tydata.yaml

```diff
@@ -0,0 +1,140 @@
+        </ul>
+    - title: Plug SkyConnect into your board
+      image: skyconnect-raspi-01.jpg
+      content: |
+        <ul>
+          <li> Plug the extension cable into your Home Assistant instance.</li>
+        </ul>
+    - title: Locate the SkyConnect integration
+      image: skyconnect-discovered-01.png
+      content: |
+        <ol>
+          <li>In Home Assistant (<a href="http://homeassistant.local:8123" target="_blank">http://homeassistant.local:8123</a>), go to <b>Settings</b>><b>Devices & Services</b>.</li>
+          <li>SkyConnect should now have been discovered.</li>
+          <li>Select <b>Configure</b>.</li>
+        </ol>
+    - title: Set up the SkyConnect integration
+      image: skyconnect-discovered-02.png
+      content: |
+        <ul>
+          <li>Select <b>Submit</b> to start the setup process.</li>
+        </ul>
+    - title: Locate the SkyConnect integration
+      image: skyconnect-discovered-03.png
+      content: |
+        <ul>
+          <li>Select <b>Create a network</b>.</li>
+        </ul>
+    - title: Add devices to area
+      image: skyconnect-discovered-04.png
+      content: |
+        <ol>
+          <li>Once the network has been set up, add the Zigbee Coorinator to an area.</li>
+          <li>Select <b>Finish</b>.
+          <ul>
+          <li>Your Zigbee devices are now added to your Zigbee network.</li>
+        </ul></li>
+        </ol>
+    - title: Setup Zigbee devices
+      image: skyconnect-discovered-05.png
+      content: |
+        <ul>
+          <li>You can start pairing the Zigbee devices to your new Zigbee network.</li>
+        </ul>
+procedureMigrateZigbee:
+  title: Migrating a Zigbee2MQTT network to SkyConnect
+  steps:
+    - title: Create a Zigbee2MQTT backup
+      image: z2m-backup-01.png
+      content: |
+        <ol>
+          <li>Under <b>Settings</b>><b>Add-ons</b>, open the <b>Zigbee2MQTT</b> add-on and select <b>Open Web UI</b>.</li>
+          <li>In the Zigbee2MQTT UI, go to <b>Settings</b>><b>Tools</b> and select <b>Request z2m backup</b>.
+            <ul>
+              <li>A zip file should now be downloaded into your <b>Downloads</b> folder.</li>
+            </ul>
+          </li>
+          
+        </ol>
+    - title: Unzip the backup folder
+      image: z2m-backup-restore-01.png
+      content: |
+        <ul>
+          <li>In your <b>Downloads</b> folder, unzip the <b>z2m-backup</b>.
+            <ul><li>It should contain a <b>coordinator_backup.json</b> file.</li></ul>
+          </li>
+        </ul>
+    - title: Uninstall the Zigbee2MQTT 
+      image: z2m-delete-add-on.png
+      content: |
+        <ul>
+          <li>Under <b>Settings</b>><b>Add-ons</b>, in the <b>Zigbee2MQTT</b> add-on, select <b>Uninstall</b>.</li>
+        </ul>
+    - title: Delete the Mosquitto broker 
```

Comment:
Let's skip this step, people can keep this around as they might use it for other integrations.

Reply:
That means the devices stay around (as they are there as persistent MQTT messages). People can manually delete the devices I guess, but if one installed the broker only to use Z2M, deleting seems better option.

How about educating people/offering both options? E.g.
- If you don't use MQTT for other purpose than Z2M, then you can savely delete the MQTT integration
- If you use MQTT for other devices, you'll have to delete the Z2M devices manually

Or do we have a better option here?